Core/source/platform/util.ts

```diff
@@ -30,3 +30,9 @@ export const setGlobals = () => {
 export const iso2022jpToUtf = (content: Buf) => {
   return dereq_encoding_japanese.convert(content, { to: 'UTF8', from: 'JIS', type: 'string' });
 }
+
+export const debugPrintArray = (title: string, arr: any[]) => {
```

Comment:
Is it used anywhere?

Reply:
I wrote this. No longer needed, I think.